aten/src/ATen/native/MetaTensor.cpp

```diff
@@ -12,22 +12,67 @@ Tensor empty_meta(
   c10::optional<bool> pin_memory,
   c10::optional<c10::MemoryFormat> memory_format
 ) {
-  // TODO: deduplicate this logic with empty_cpu
+
+  TORCH_INTERNAL_ASSERT_DEBUG_ONLY(device_or_default(device).type() == DeviceType::Meta);
+  // NB: because there is no SparseMeta (yet), non-strided layout is
+  // exerciseable
+  TORCH_CHECK_NOT_IMPLEMENTED(
+    layout_or_default(layout) == Layout::Strided,
+    "strided meta tensors not supported yet"
```

Comment:
This is saying to error if the layout isn't strided, right? Shouldn't the error be "non-strided meta tensors are not supported yet"?

Reply:
yup